lib/services/ios-log-filter.ts

```diff
@@ -41,14 +41,16 @@ export class IOSLogFilter extends iOSLogFilterBase.IOSLogFilter implements Mobil
 				// This code removes unnecessary information from log messages. The output looks like:
 				// CONSOLE LOG file:///location:row:column: <actual message goes here>
 				if (pid) {
-					const searchString = "[" + pid + "]: ";
-					const pidIndex = line.indexOf(searchString);
-					if (pidIndex > 0) {
-						line = line.substring(pidIndex + searchString.length, line.length);
+					const pidIndex = line.indexOf(`[${pid}]`);
+					const ios11PidIndex = line.indexOf(` ${pid} `);
+					if (pidIndex > 0 || ios11PidIndex > 0) {
+						const pidRegex = new RegExp(`.*[\\[\\s]{1}${pid}[\\s\\]]{1}.*?:(?:.*?\\(NativeScript\\) )*`);
+						line = line.replace(pidRegex, "").trim();
 						this.getOriginalFileLocation(line);
 						result += this.getOriginalFileLocation(line) + "\n";
-						continue;
 					}
+
+					continue;
```

Comment:
If we don't match with neither `pidIndex` nor `ios11PidIndex` I do not believe we'd want to `continue`.
I think this `continue` statement should be returned in the previous block

Reply:
Taking `continue` outside the `if` statement prevents lines that don't contain the pid to be logged.